packages/gatsby/src/redux/actions/public.js

```diff
@@ -1424,16 +1424,20 @@ actions.createServerVisitedPage = (chunkName: string) => {
  * @param {Object} manifest Manifest data
  * @param {string} manifest.manifestId An id which ties the revision unique state of this manifest to the unique revision state of a data source.
  * @param {Object} manifest.node The Gatsyby node to tie the manifestId to. See the "createNode" action for more information about the node object details.
+ * @param {string} manifest.updatedAtUTC (optional) The time in which the node was last updated
```

Comment:
This should also mention the specific format it should be in (e.g. with timezone?

Reply:
The format of this depends on the format that the CMS uses to store this value, as long as it's a valid date string it should be fine.